src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchCriteriaPactTest.java

```diff
@@ -360,6 +368,57 @@ public Task createTaskForTaskTypeSearch() {
         );
     }
 
+    public Task createTaskForTaskStateSearch() {
```

Comment:
Naming here could be better.  A Task search using the State does not have to return Completed Tasks.  It could be searching for any state.